oommfc/drivers/driver.py

```diff
@@ -211,7 +211,7 @@ def drive(self, system, /, dirname='.', append=True, fixed_subregions=None,
                         and callable(term.func)):
                     del term.tlist
                     del term.dtlist
-                    del mif
+            del mif
```

Comment:
Is it necessary to delete those here? Can't we rely on garbage collection? Also, these variables should be very small anyway.

Reply:
I can remove these deletions for now. They are technically not required.

But we should start thinking about memory consumption during the OOMMF run in general.